ttnn/ttnn/operations/core.py

```diff
@@ -160,7 +160,10 @@ def from_torch(
     mesh_mapper: Optional[ttnn.TensorToMesh] = None,
 ) -> ttnn.Tensor:
     """
-    Converts the `torch.Tensor` tensor into a `ttnn.Tensor`.
+    Converts the `torch.Tensor` tensor into a `ttnn.Tensor`. For bfloat8_b or bfloat4_b format, call from_torch twice, first call
```

Comment:
This is not clear.

Are you trying to say that this code calls itself twice? Are you trying to say that the user should call this code twice?

I think this needs reworking.

Also, I'm wondering if the comment
```
It does not call to_layout for bfloat8_b or bfloat4_b
```
shouldn't be applicable here.

Reply:
yeah it calls itslef twice, 
"It does not call to_layout for bfloat8_b or bfloat4_b" - during  tensor creation it already called converted layout, so no need to convert twice

Reply:
I read
> For bfloat8_b or bfloat4_b format, call from_torch twice
I think it means call me twice.
But I know this is not what you want.

Do we really need this detail of implementation in the doc?

Reply:
think it's good to have? this function is not easy to read as it calls itself twice. probably users want to be aware of this.